Dockerfile

```diff
@@ -52,6 +52,10 @@ FROM base-image AS install
 RUN mkdir ~/carma_ws
 COPY --from=source-code --chown=carma /home/carma/src /home/carma/carma_ws/src
 
+# add astuff messages
+WORKDIR /home/carma/carma_ws/src
+RUN git clone https://github.com/astuff/astuff_sensor_msgs -b melodic
```

Comment:
I think this clone would be better placed in the docker/checkout.sh script to match the pattern our team has been using.

Reply:
Fixed.